salt/modules/omapi.py

```diff
@@ -13,7 +13,7 @@
 '''
 
 # Import python libs
-from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, print_function
```

Comment:
This needs a comment as to why we don't have `unicode_literals` imported, so that someone doesn't add it back later.